libpod/config.go

```diff
@@ -1,4 +1,4 @@
-package libkpod
+package libpod
 
 import (
```

Comment:
This file really needs a header comment of some kind saying it is deprecated and going away as soon as we can find the time to remove it